checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/configuration/CardConfigurationClientFactory.kt

```diff
@@ -1,15 +1,39 @@
 package com.worldpay.access.checkout.api.configuration
 
+import com.worldpay.access.checkout.Card
+import com.worldpay.access.checkout.api.Callback
+import com.worldpay.access.checkout.logging.LoggingUtils
+import com.worldpay.access.checkout.model.CardConfiguration
+import com.worldpay.access.checkout.validation.AccessCheckoutCardValidator
+
 /**
  * Creates a [CardConfigurationClient] instance
  */
-object CardConfigurationClientFactory {
+internal object CardConfigurationClientFactory {
 
     /**
      * @return a [CardConfigurationClient] instance
      */
     @JvmStatic
-    fun createClient(): CardConfigurationClient {
+    internal fun createClient(): CardConfigurationClient {
         return CardConfigurationClientImpl()
     }
+}
+
+object CardConfigurationFactory {
```

Comment:
I'd probably move this object to it's own file, as it's mixed in with the `CardConfigurationClientFactory`. Either that or rename this whole file to be `CardConfigurationFactories`.

Reply:
👍